tests/testthat/test-tags.r

```diff
@@ -526,6 +526,37 @@ test_that("NA attributes are rendered correctly", {
   )
 })
 
+test_that("Tag list tree is rendered in DOM tree order", {
+  # Tree order is preorder, depth-first traversal
+  # https://dom.spec.whatwg.org/#concept-tree
+  #
+  # Test for depth-first (not breadth-first)
+  # https://stackoverflow.com/a/16113998/1583084
+  expect_identical(
```

Comment:
I don't see how this particular test checks how the tree is traversed. I think this test can be removed because the other test does check how the tree is traversed.

Reply:
Example of how tree traversal order shouldn't affect the output in a case like this:
https://gist.github.com/wch/5e4a5df737d0fbe1effca5621bea8dbb

Reply:
Right -- I suppose this isn't testing `tagify()`/`rewriteTags()` tree traversal, but it is testing `flattenTags()`'s tree flattening algorithm.

Anyway, I'm happy to remove it since we have plenty other tests that will make sure that `flattenTags()` is depth-first